tests/bin/install-wp-tests.sh

```diff
@@ -149,21 +150,25 @@ install_db() {
 }
 
 install_woocommerce() {
+	WOO_INSTALL_VERSION=$WOO_VERSION
+	echo "Woo Version: $WOO_VERSION"
+	if [ $WOO_VERSION == 'latest' ] ; then
+		WOO_INSTALL_VERSION=latest-stable
+	fi
+	echo "Woo Install Version: $WOO_INSTALL_VERSION";
 	# get built plugin from .org
-	download https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip "$TMPDIR/woocommerce.zip"
+	download "https://downloads.wordpress.org/plugin/woocommerce.$WOO_INSTALL_VERSION.zip" "$TMPDIR/woocommerce.zip"
 	unzip -q $TMPDIR/woocommerce.zip -d "$WP_CORE_DIR/wp-content/plugins"
 
-	# Script Variables
-	BRANCH=$TRAVIS_BRANCH
-	REPO=$TRAVIS_REPO_SLUG
-
-	# Get github version
-	if [ "$TRAVIS_PULL_REQUEST_BRANCH" != "" ]; then
-		BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
-		REPO=$TRAVIS_PULL_REQUEST_SLUG
+	git clone "https://github.com/woocommerce/woocommerce.git" "$TMPDIR/woocommerce-git"
```

Comment:
Unfortunately the entire repo needs cloned to support checking out specific tags. This impacts the run time for the tests by ~1-2min (depending on network connection) and also the bytes transferred. Ideally we'll find a different solution for this but for now this helps unblock failing tests.